warehouse/query-core/src/main/java/datawave/query/jexl/visitors/FieldToFieldComparisonVisitor.java

```diff
@@ -0,0 +1,69 @@
+package datawave.query.jexl.visitors;
+
+import datawave.query.jexl.JexlASTHelper;
+import org.apache.commons.jexl2.parser.*;
+
+public class FieldToFieldComparisonVisitor extends RebuildingVisitor {
```

Comment:
Short description for what this visitor does, e.g. "will wrap field to field comparison nodes in evaluation only markers", and perhaps an example "FIELD_A == FIELD_B" -> "(\_eval\_) & (FIELD_A.... etc